docs/development.md

```diff
@@ -64,9 +60,19 @@
 
 ## Development notes
 
+- Set version numbers:
```

Comment:
Is it mandatory or optional?

Reply:
for dev, it should be, otherwise, you'll end up with 'latest'
thoughts?

Reply:
fixed, added "optional" information